build/configure

```diff
@@ -389,7 +389,7 @@ ruby_configure_quals=(
      )
 
 ruby_config_quals_sysins="--prefix=/opt/microsoft/omsagent/ruby"
-ruby_config_quals_testins="--prefix=/usr/local/ruby-2.2.0c"
+ruby_config_quals_testins="--prefix=${base_dir}/intermediate/local_ruby-2.2.0c"
```

Comment:
No need to have the letter separator. Just call it local-ruby-2.2.0 at that point, since any chances to it would just rebuild it anyway. Also, rather than putting it in a fixed dir (ignoring debug vs. release), shouldn't this be based on $BUILD_CONFIGURATION, which is what (I believe) the subdirectory is called under intermediate?

This will likely require "sudo" to do a "git nuke", right, since Ruby will be "installed" into the intermediate_dir with root permissions? This will be slightly less convenient for OMS devs that use 'git nuke', but it looks like the build system uses sudo for that already.